src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/errors.py

```diff
@@ -81,3 +81,10 @@ class NoAccountsFoundError(Exception):
     """
 
     pass
+
+class WaveSizeInsufficientError(Exception):
```

Comment:
NIT: I would rename this to 
```suggestion
class InsufficientWaveSizeError(Exception):
```